app/controllers/miq_ae_class_controller.rb

```diff
@@ -2749,4 +2752,28 @@ def process_element_destroy_via_queue(element, klass, name)
                {:model => model_name, :name => record_name, :error_msg => bang.message}, :error)
     end
   end
+
+  def breadcrumbs_options
+    {
+      :breadcrumbs => [
+        {:title => _("Automation")},
+        {:title => _("Automate")},
+        {:title => _("Explorer")},
+      ],
+    }
+  end
+
+  def accord_name
+    features.each do |feature|
+      return feature.title if feature.accord_name == x_active_accord.to_s
+    end
+    nil
+  end
+
+  def accord_container
+    features.each do |feature|
```

Comment:
If you want to return something from an `Enumerable` you don't want to use `each`, there's always a better method:
```ruby
features.find { |f| f.accord_name == x_active_accord.to_s }.try(:container )
```